### PR TITLE
[SYCL][DOC] Add notes for low level runtime setup/installation

### PR DESCRIPTION
**Pull request by v-klochkov**
Signed-off-by: Vyacheslav N Klochkov <vyacheslav.n.klochkov@intel.com>

### CODE REVIEW

File: sycl/doc/GetStartedWithSYCLCompiler.md [hunk 1]
```diff
@@ -117,25 +117,42 @@ should be used.
 To run SYCL  applications on OpenCL devices, OpenCL implementation(s) must be
 present in the system.
 
-To execute SYCL tests on Intel devices following runtimes should be installed:
+The following `GPU` runtimes should be installed to run SYCL applications on  Intel `GPU` devices:
```
Comment (bader) [inline sycl/doc/GetStartedWithSYCLCompiler.md]: Please, limit line length to 80 characters.

File: sycl/doc/GetStartedWithSYCLCompiler.md [hunk 2]
```diff
@@ -117,25 +117,42 @@ should be used.
...
-* OpenCL&trade; runtime for Intel GPU [Intel&reg; Download
+* Windows: [Intel&reg; Download
    Center](https://downloadcenter.intel.com/product/80939/Graphics-Drivers)
 
-* The experimental Intel&reg; CPU Runtime for OpenCL&trade; Applications with
-  SYCL support on Windows will be available soon.
 
-Please, refer to [the Release Notes](../ReleaseNotes.md) for recommended Intel
-runtime versions.
+To install the OpenCL runtime for Intel `CPU` and `FPGA` devices the corresponding runtime asset/archive should be downloaded from [SYCL Compiler and Runtime updates](../ReleaseNotes.md) and installed using the following procedure.
```
Comment (bader) [inline sycl/doc/GetStartedWithSYCLCompiler.md]:
```suggestion
To install Intel `CPU` and `FPGA` runtimes for OpenCL devices the corresponding runtime asset/archive should be downloaded from [SYCL Compiler and Runtime updates](../ReleaseNotes.md) and installed using the following procedure.
```

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 1 above)
Comment (bader) [inline sycl/doc/GetStartedWithSYCLCompiler.md]: The links below are not to exact version of the runtime, these are the links to the web-pages where Linux/Windows versions of GPU runtime can be downloaded.
I think we have to update the wording to make it clear.
NOTE: recommended version should separately provided in addition to these links.

File: sycl/doc/GetStartedWithSYCLCompiler.md [hunk 3]
```diff
@@ -117,25 +117,42 @@ should be used.
...
+
+**Linux**
+
+1) Extract the archive:
+```bash
+mkdir -p /opt/intel/oclcpuexp
+cd /opt/intel/oclcpuexp
+tar -zxvf oclcpuexp-2019.8.7.0.0725_rel.tar.gz
```
Comment (bader) [inline sycl/doc/GetStartedWithSYCLCompiler.md]: I think we should avoid using exact name of the package.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 1 above)
Comment (romanovvlad) [inline sycl/doc/GetStartedWithSYCLCompiler.md]: plural "runtimes" is a little bit confusing as actually you need to install only one GPU runtime.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 2 above)
Comment (romanovvlad) [inline sycl/doc/GetStartedWithSYCLCompiler.md]: Not sure the instruction below applicable for OpenCL runtime for FPGA device.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 1 above)
Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Ok, will change: runtimes -> runtime

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 2 above)
Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Ok. changed.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 1 above)
Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Fixed.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 2 above)
Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Well, If you are not sure, then I remove FPGA.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 1 above)
Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Ok, I re-phrased to the followinig:

> The `GPU` runtime that is needed to run SYCL application on Intel `GPU` devices
> can be downloaded from the following web pages:

I also do not understand what you mean by this: "recommended version should separately provided in addition to these links.".
How I can recommend without knowing what user has. For example, here: https://github.com/intel/compute-runtime/releases all recommendations are on that page and there is no need to recommend something here on this Gettin Started Page.

Comment (bader) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Current version of the [document](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedWithSYCLCompiler.md#install-low-level-runtime) has quite important information, removed by this patch:
```
Please, refer to the [the Release Notes](../ReleaseNotes.md) for recommended Intel runtime versions.
```
As of today there are 78 release of NEO runtime available at https://github.com/intel/compute-runtime/releases.
Unfortunately, not all of these versions are compatible with the latest version of SYCL implementation.

We should guide our users and specify "working" versions of GPU runtime.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 3 above)
Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Ok, I changed this place.

**Comment by v-klochkov** [comment]
Thank yo for the comments!
I have just updated the patch. Please re-review.

File: sycl/doc/GetStartedWithSYCLCompiler.md (hunk 1 above)
Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Ok, I will restore the reference to Release Notes.

Regarding the recommendations and what version are working..., even I don't know, which are working and which are not. So, I don't seem a right person for writing that specifics here.

Comment (pvchupin) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Basically idea is that we should have instructions here without specific details on the timestamps. Or we should say clearly that timestamps given are just an examples.
Every month we are publishing release notes which has recommended timestamp per target. You can also mention this here.

Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: In my opinion restoring this statement is more than enough:
"Please, refer to [the Release Notes](../ReleaseNotes.md) for recommended Intel runtime versions."

Comment (pvchupin) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Probably yes.

**Comment by v-klochkov** [comment]
@bader the patch-update has fixes for you recent comments.
Please merge this PR or give more comments.

File: sycl/doc/GetStartedWithSYCLCompiler.md [hunk 4]
```diff
@@ -117,25 +117,57 @@ should be used.
...
-runtime versions.
+To install Intel `CPU` runtime for OpenCL devices the corresponding runtime
+asset/archive should be downloaded from
+[SYCL Compiler and Runtime updates](../ReleaseNotes.md) and installed using
+the following procedure.
+
+**Linux**
+
+1) Extract the archive. For example, for archive with the name
+'oclcpu_rt_new_version.tar.gz' you would run the following commands
```
Comment (bader) [inline sycl/doc/GetStartedWithSYCLCompiler.md]: I would use replace `oclcpu_rt_new_version` with `oclcpu_rt_<version>` everywhere.

Comment (v-klochkov) [reply sycl/doc/GetStartedWithSYCLCompiler.md]: Ok, I have just force-pushed the requested fix. Thank you.

**Pull request merged**